tests-legacy/Unit/ContextMocker.php

```diff
@@ -101,7 +101,7 @@ public function mockContext()
         Phake::when($context->customer)->getGroups()->thenReturn(array());
         $context->cookie   = Phake::mock('Cookie');
         $context->country  = Phake::mock('Country');
-        $context->language = new Language((int) Configuration::get('PS_LANG_DEFAULT'));
+        $context->language = new Language((int) Configuration::get('PS_DETECT_LANG'));
```

Comment:
Why this change?

Reply:
There is no value in database for `PS_LANG_DEFAULT`

Reply:
I knew it was weird. I try to restore the old value. Waiting for ci.